docs/modules/json.md

```diff
@@ -0,0 +1,110 @@
+[< Back](.)
+
+json
+================
+
+The `json` module provides a simple interface to read and write [json files and strings](https://www.json.org/json-en.html).
+
+It contains the following classes:
+
+* [Json](#json)
+* [JsonOptions](#jsonoptions)
+* [JsonError](#jsonerror)
+
+## Json
+
+### Constructor
+
+#### `constructor parse(value:String, options:Num)`
+This returns a new `Json` object .
+
+### Instance Properties
+
+#### `json: Map`
+This returns the _Map_ holding the information parsed from the initial json string.
+
+### `options: Num`
+The options used when creating this object.
+
+### `raw: String`
+This returns the initial _String_ provided before parsing.
+
+### `error: JsonError`
+If a parsing error is found this will contain the line, position and error message.
+
+### Instance Methods
+
+### `dumps(): String`
+This will transform the _json_ property to _String_. with default options.
+
+### `dumps(options: Num): String`
+This will transform the _json_ property to _String_. With custom options. 
+
+### `save(path: String)`
+This will execute `dumps()` and then save the result to a file specified in `path`.
+
+### `save(path: String, options:Num)`
+This will execute `dumps(options)` and then save the result to a file specified in `path`.
+
+### Static Methods
+
+#### `static parse(value:String): Json`
+Returns a new _Json_ object with default options.
+
+#### `static fromFile(path:String): Json`
+Reads the contents of a file in `path` and returns a new _Json_ object with default options.
+
+#### `static fromFile(path:String, options:Num): Json`
```

Comment:
I prefer to combine overloaded methods, and describe the defaults, rather than separate entries for each.
```
#### `static fromFile(path: String): Json`
#### `static fromFile(path: String, options: Num): Json`
Reads the contents of the file located at `path` and returns a new Json object with the specified options. If no options are provided, `JsonOptions.NIL` is assumed.
```